scholarly/publication.py

```diff
@@ -254,7 +254,7 @@ def fill(self):
 
     @property
     def citedby(self) -> _SearchScholarIterator or list:
-        """Searches GScholar for other articles that cite this Publication and
+        """Searches Google Scholar for other articles that cite this Publication and
```

Comment:
This was to make sure that the code is PEP-8 Compliant.
Is it still?

Reply:
Hm, I am not sure exactly how this is related to PEP-8 since I perceived that as a human-readable comment only. Can you please be more specific?